api/feedback/v1/feedback.proto

```diff
@@ -94,14 +92,8 @@ message SubmitFeedbackRequest {
   // TODO: remove if we decide to record feedback only when a user clicks the submit button
   string id = 1 [ (validate.rules).string = {min_bytes : 1} ];
   Feedback feedback = 2 [ (validate.rules).message = {required : true} ];
+  FeedbackMetadata metadata = 3 [ (validate.rules).message = {required : true} ];
```

Comment:
changed it to not be nested for convenience reasons since the feedback and metadata are stored in diff columns.